class.jetpack.php

```diff
@@ -639,7 +639,9 @@ private function __construct() {
 	}
 
 	function point_edit_links_to_calypso( $default_url, $post_id ) {
-		if ( ! $post = get_post( $post_id ) ) {
```

Comment:
It'll be nice to sync this change to WP.com.

Reply:
Good point, after we merge this I'll create a janitorial diff to sync the new changes there.